motr/ut/cs_ut_main.c

```diff
@@ -703,6 +703,14 @@ static void test_cs_ut_iem(void)
 		M0_MOTR_IEM_EVENT_FREE_SPACE,
 		"DUMMY Description, Dummy number[%d], Dummy text [%s]",
 		__LINE__, __FUNCTION__);
+
+	m0_iem(__FILE__, __FUNCTION__, __LINE__, \
```

Comment:
No need for the back slash at the end of this line.

Reply:
This message is not needed and appeared here because I did not check the ut code. I will remove this m0_iem call.